controllers/JobController.js

```diff
@@ -67,59 +25,80 @@ exports.JobListByApplicantId = [
  * @returns {Object}
  */
 exports.JobList = [
-	auth,
-	function (req, res) {
-		try {
-			JobModel.find().then((Jobs) => {
-				if (Jobs.length > 0) {
-					let JobData = [];
-					Jobs.forEach(job => {
-						let data = new jobData(job);
-						JobData.push(data);
-					});
-					return apiResponse.successResponseWithData(res, "Operation success", JobData);
-				} else {
-					return apiResponse.successResponseWithData(res, "Operation success", []);
-				}
-			});
-		} catch (err) {
-			//throw error in json response with status 500.
-			return apiResponse.ErrorResponse(res, err);
-		}
-	}
+  auth,
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 1 tab but found 2 spaces.](https://app.codacy.com/gh/n4alphapesto/hirebook-backend/pullRequest?prid=7771833)
```suggestion
	auth,
```